api.web.yaml

```diff
@@ -76,3 +90,26 @@ components:
         agreement_id:
           type: string
           description: UUID of the agreement that has been agreed to
+
+    ConfirmedAgreement:
```

Comment:
naming suggestions welcomed!

Reply:
maybe signed instead of confirmed? but I think this is also fine